src/app/main.cpp

```diff
@@ -216,13 +206,6 @@ int main(int argc, char *argv[])
         app->setAttribute(Qt::AA_DontShowIconsInMenus);
 #endif
 
-#ifndef DISABLE_GUI
-        if (!upgrade()) return EXIT_FAILURE;
-#else
-        if (!upgrade(!params.shouldDaemonize
-                     && isatty(fileno(stdin))
-                     && isatty(fileno(stdout)))) return EXIT_FAILURE;
-#endif
```

Comment:
Maybe keep it for further use? Just to clear "upgrade()"?

Reply:
No, I mean call to noop "upgrade()" in this place. When we re-need some upgrade code we just put it there (in  "upgrade()").